modules/45_carbonprice/diffLin2Lin/datainput.gms

```diff
@@ -14,7 +14,7 @@
 ***                                         carbon price of developed countries increases linearly with fixed annual increase given by cm_taxCO2inc_after_peakBudgYr (default = 0, i.e. constant)
 ***       (with iterative_target_adj = 5):  carbon price of developed countries keeps increasing linearly (with same slope) until end of century, i.e. no change after peak year 
 ***       (with iterative_target_adj = 0):  after year given by cm_peakBudgYr (default = 2050), carbon price of developed countries increases linearly with fixed annual increase given by cm_taxCO2inc_after_peakBudgYr (default = 0, i.e. constant),
-***                                         for linearly increasing carbon price (with same slope) until end of century, set cm_peakBudgYr = 2100         
+***                                         for linearly increasing carbon price (with same slope) until end of century, set cm_peakBudgYr = 2110         
```

Comment:
Consistent implementation in (diffLin2Lin) and (diffExp2Lin): If linear/exponential increase is selected until end of century (e.g. with cm_iterative_target_adj = 5), carbonprices are set constant from 2110 onward.